pvlib/test/data/gold/test_sdm.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pprint
+import uuid
+
+from numpy.testing import assert_allclose
+
+from conftest import requires_interval, requires_scipy
+from pvlib import pvsystem
+from pvlib.data.gold import sdm
+
+
+def test_sum_current():
+    # Test sum_current computation against comitted gold dataset
+    gold_dataset = sdm.convert_gold_dataset(
+        sdm.load_gold_dataset(json_rel_path=os.path.join(
+            os.path.dirname(sdm.__file__), "sdm.json")))
+
+    for device in gold_dataset["devices"]:
+        for iv_curve in device["iv_curves"]:
+            current_sum_res = sdm.sum_current(
+                iv_curve["r_sh"], iv_curve["r_s"], iv_curve["nNsVth"],
+                iv_curve["i_gold"], iv_curve["v_gold"], iv_curve["i_0"],
+                iv_curve["i_l"])
+
+            # atol typically smaller that the interval tolerance used for gold
```

Comment:
Maybe I am confused but I thought ``atol`` would need to be larger than the interval tolerance. Won't the test fail if ``atol`` is smaller than the interval tolerance? Also not sure sure quite what you mean by "typically" -- is there another failure mode to be concerned about?

Reply:
Interval computations are conservative, so the intervals should be no larger than any specific residual. In this case we have the same computed quantity (V + I*Rs) showing up twice in the current sum residual computation, and this in known to cause the intervals to be "pessimistic" because the two quantities are treated independently in the overall computation, which they are not. I updated the comment here to hopefully be less ambiguous.